packages/neo-one-client/src/DeveloperClient.ts

```diff
@@ -1,9 +1,16 @@
-import { DeveloperProvider, Options } from './types';
+import { NEOONEProvider } from './provider';
+import { ContractParameter, DeveloperProvider, Options, UserAccountID } from './types';
+import { LocalKeyStore, LocalUserAccountProvider } from './user';
 
 export class DeveloperClient {
   private readonly developerProvider: DeveloperProvider;
+  private readonly localUserAccountProvider: LocalUserAccountProvider<LocalKeyStore, NEOONEProvider>;
 
-  public constructor(developerProvider: DeveloperProvider) {
+  public constructor(
+    developerProvider: DeveloperProvider,
+    localUserAccountProvider: LocalUserAccountProvider<LocalKeyStore, NEOONEProvider>,
```

Comment:
This should use the same structure as `Client` (i.e. an object of `UserAccountProvider`s not a specific implementation)